lms/templates/register.html

```diff
@@ -75,7 +84,7 @@
                 isValid: false,
                 message: "${_('Your legal name must be a minimum of two characters long')}",
                 field: "${_('Full Name')}",
-                pattern: /^[a-zA-ZàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŽ∂ð ,.'-]{2,255}$/,
+                pattern: /^[a-zA-ZàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŚŽ∂ð ,.'-]{2,255}$/,
```

Comment:
jeśli ta zmiana miała dodać Ś, to brakuje jeszcze małego ś

Reply:
naprawione w hotfix 1.28.8